crates/physical-plan/src/plan.rs

```diff
@@ -44,13 +69,19 @@ pub struct IndexScan {
 
 /// BTrees support equality and range scans
 pub enum IndexOp {
-    Eq(AlgebraicValue, TyId),
-    Range(Bound<AlgebraicValue>, Bound<AlgebraicValue>, TyId),
+    Eq(AlgebraicValue),
+    Range(Bound<AlgebraicValue>, Bound<AlgebraicValue>),
```

Comment:
I think this deserve a type, so is more explicit how we convert from `Ops` to this, ie:: 

```rust
struct BoundsOp {
   from:Bound<AlgebraicValue>, 
   to: Bound<AlgebraicValue>
}
```